app/MMB/src/scenes/timeline/event/Event.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet } from '@kiwicom/mobile-shared';
+
+import IconVerticalLine, {
+  type IconVerticalLineType,
+} from './IconVerticalLine';
+import TimeLocation, { type TimeLocationType } from './TimeLocation';
+import DescriptionComment, {
+  type DescriptionCommentType,
+} from './DescriptionComment';
+
+export type Props = {|
+  ...IconVerticalLineType,
+  ...TimeLocationType,
+  ...DescriptionCommentType,
+  first: boolean,
+|};
+
+const styles = StyleSheet.create({
+  event: {
+    flexDirection: 'row',
+    paddingStart: 5,
+    paddingEnd: 5,
+  },
+});
+
+const TOP_OFFSET_FIRST = 30;
+const TOP_OFFSET_NORMAL = 5;
+
+export default function Event(props: Props) {
+  const { dateTime, location, icon, description, comment, first } = props;
+  const topOffset = first ? TOP_OFFSET_FIRST : TOP_OFFSET_NORMAL;
```

Comment:
This is one of the props that are most probably very wrong. Did you see the video yesterday? 🙂 Are we going to create `last` because last is going to be different? And `odd`, `even`, `special`, ...

It does not sound right.